tmk_core/protocol/vusb/main.c

```diff
@@ -27,6 +27,10 @@
 #include "wait.h"
 #include "sendchar.h"
 
+#ifdef EEPROM_DRIVER
+#    include "eeprom_driver.h"
+#endif
+
```

Comment:
Another part of EEPROM code which does not seem to belong here.

Reply:
See my reply in lufa.c.